build-cmd.sh

```diff
@@ -271,20 +271,19 @@ case "$AUTOBUILD_PLATFORM" in
     PREFIX="$STAGING_DIR"
 
     opts="-m$AUTOBUILD_ADDRSIZE $LL_BUILD_RELEASE"
-
+    opts=$(echo $opts | sed -e 's/-std=c++17//')
```

Comment:
The `remove_cxxstd` shell function from [secondlife/build-variables](https://github.com/secondlife/build-variables/blob/master/functions#L79-L85) removes that switch regardless of the specific version.

Reply:
Would this be expected to be called automagically or does it need to be called manually? I suppose the latter, as it did not kick in for me.
And I did not try to call it myself (I was not aware of this function). > Will try to switch over to it and then update the  PR accordingly